runtime/kreivo/src/config/revive/mod.rs

```diff
@@ -0,0 +1,41 @@
+use super::*;
+use pallet_assets::precompiles::{InlineIdConfig, ERC20};
+
+parameter_types! {
+	pub const DepositPerItem: Balance = deposit(1, 0);
+	pub const DepositPerByte: Balance = deposit(0, 1);
+	pub CodeHashLockupDepositPercent: Perbill = Perbill::from_percent(30);
+	pub const ChainId: u64 = 2281;
+}
+
+pub type TrustBackedAssetsInstance = pallet_assets::Instance1;
+// pub type PoolAssetsInstance = pallet_assets::Instance3; // Should be created a new instance for this (Instance3)?
+pub type PoolAssetsInstance = pallet_assets::Instance1;
+use frame_system::EnsureSigned;
+
+impl pallet_revive::Config for Runtime {
+	type Time = Timestamp;
+	type Currency = Balances;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
+	type DepositPerItem = DepositPerItem;
+	type DepositPerByte = DepositPerByte;
+	type WeightPrice = pallet_transaction_payment::Pallet<Self>;
+	type WeightInfo = pallet_revive::weights::SubstrateWeight<Self>;
+	type Precompiles = (
+		ERC20<Self, InlineIdConfig<0x120>, TrustBackedAssetsInstance>,
+		ERC20<Self, InlineIdConfig<0x320>, PoolAssetsInstance>,
+	);
+	type AddressMapper = pallet_revive::AccountId32Mapper<Self>;
+	type RuntimeMemory = ConstU32<{ 128 * 1024 * 1024 }>;
+	type PVFMemory = ConstU32<{ 512 * 1024 * 1024 }>;
+	type UnsafeUnstableInterface = ConstBool<false>;
+	type UploadOrigin = EnsureSigned<Self::AccountId>;
```

Comment:
Initially should be permissioned so let's do the [same as contrats](https://github.com/virto-network/kreivo/blob/master/runtime/kreivo/src/config/contracts/mod.rs#L106C2-L106C72)